modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -167,3 +169,25 @@ main `react` package.
 - `Box` was promoted from `@workday/canvas-kit-labs/common` to `@workday/canvas-kit-react/layout`
 - `Flex` was promoted from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
 - `Stack` was promoted from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
+
+## Tokens Updates
+
+### Depth Tokens
+
+**Changes:**
+
+- Depth `inset` was removed from token list.
+- Added new depth: `none` (to remove default depth from component), `5` and `6`.
+- Now all depth will add only `box-shadow` with 2 shadows style and no border.
+
+Also we changed default depth for the next components:
+
+- Card: Depth 2 → Depth 1
+- Select Menu, Menu: Depth 2 → Depth 3
+- Breadcrumbs Dropdown Menu: Depth 2 → Depth 3
+- Combobox: Depth 1 → Depth 3
+- Popup: Depth 2 → Depth 5
+- Dialog: Depth 2 → Depth 5
+- Color Picker Palette, Toast: Depth 2 → Depth 5
+- Modal: Depth 2 → Depth 6
+- Side Panel (alternate variant): Depth 3 → Depth 6
```

Comment:
This will be really helpful for teams when they do visual regression testing. Thank you!

Reply:
There's also a Side Panel in the main package. Maybe this should say "Preview Side Panel" to be more clear?